main.js

```diff
@@ -0,0 +1,33 @@
+const menuIconMobile = document.querySelector('.navbar__hamb');
+const menu = document.querySelector('.navbar__menu');
+const viewport = document.querySelector('body');
+const menuLinks = document.querySelectorAll('.navbar__menu__link');
+
+function isActiveClassExisted(element) {
+  return element.classList.contains('active');
+}
+
+function refreshIcon() {
+  if (isActiveClassExisted(menu)) {
+    menuIconMobile.setAttribute('src', './images/hamburguer-close.png');
+  } else {
+    menuIconMobile.setAttribute('src', './images/hamburguer.png');
+  }
+}
+
+function openMenu() {
+  menu.classList.toggle('active');
+  viewport.classList.add('blockover');
+  refreshIcon();
+}
+
+function closeMenu() {
+  menu.ClassList.remove('active');
+  viewport.classList.remove('blockover');
```

Comment:
- [ ] The letter c in classList suppose to be lowercase, not uppercase in line 25 So please fix that because now it's not removing the active class.

- [ ] Nice job fixing the navigation links in the mobile menu now when you click it takes you to the right section of the page but the scroll behavior is still fixed when you do that. So please fix that.

[Hint]: you can fix that by simply adding `overflow-y: scroll;` to the body styling in `main.css`
![image](https://user-images.githubusercontent.com/68796232/162203680-8ce0bea8-8d8c-411d-ab43-6401c869e08b.png)

Reply:
Immensely grateful for the encouraging review and helpful tips!@Omar-muhamad
Changes have been effected as suggested. Thanks a lot. Can I proceed to submit for approval now? This is my last submission chance. Looking forward to an approval next.

Reply:
Hi @ONJoseph now everything works as expected ask for another review and i think it will be approved.